snowflake_queries/snowflake_scim_token_created.yml

```diff
@@ -0,0 +1,15 @@
+AnalysisType: scheduled_rule
+Filename: snowflake_scim_token_created.py
+RuleID: Snowflake.SCIMTokenCreated
+Description: Alerts when a SCIM token is created
+DisplayName: Snowflake SCIM Access Token Created
+Enabled: false
+ScheduledQueries:
+  - Query.Snowflake.SCIMTokenCreated
+Severity: Info
+Tests:
+  -
+    Name: SCIM Token Created
+    ExpectedResult: true
+    Log:
+      query_text: "system xyz generate_scim_access_token"
```

Comment:
same question: is this the expected output from the query?

Reply:
Changed based on our conversation